internal/store/builder.go

```diff
@@ -40,7 +41,6 @@ import (
 	vpaclientset "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/client/clientset/versioned"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
-	"k8s.io/klog/v2"
```

Comment:
Could you revert this change to keep the imports sorted?

Reply:
done.